docs/randomblind.md

```diff
@@ -0,0 +1,139 @@
+
+```
+irma session --issue irma-demo.stemmen.stempas=test,test.com,14-12-2020,15-12-2020
+```
+
+Upon scanning the QR code, the user is asked permission to perform the session and
+obtain this credential.  During the session, the issuer and user decide on the
+value of the randomblind attributes, in this case the `votingnumber` attribute.
+We explain in detail how this comes to pass in the next section. The final
+value, which is at most 256 bits, ends up in the credential. The app displays
+this attribute in a [base62 representation](https://en.wikipedia.org/wiki/Base62).
+
+
+Before issuance               |  After issuance
+:----------------------------:|:----------------------------:
+![Permission](assets/rb_permission.jpg) | ![Card](assets/rb_card.jpg)
+
+## Cryptography
+In this section we will explain in detail how the issuer and the user are able
+to perform such an issuance session. Randomblindness of attributes only concerns
+the issuance protocol. The disclosure and signature protocols of involving radomblind attributes
+is in every aspect similar to disclosing or signing using a normal attribute.
+
+We can assume that both parties have 
+a scheme in which the indices of the randomblind attributes within the
+credentialtype coincide. If this is not the case, either party cancels the
+session. In the following example, we only have one randomblind attribute, but
+the protocol can be extended straightforwardly to issue multiple randomblind
+attributes.
+
+TODO: link to intro on CL-signatures.
+
+Variables $R, S, Z$ are public parameters.
+
+The issuance goes as follows:
+- The user starts by a sampling $v'$ and a random 255-bit integer $m_{r}'$ at
+  the index $r$ of the randomblind attribute.
+  This so-called *share* of the attribute remains secret, similar to
+  how the user's secret key remains secret during issuance. 
+  The user computes the commitment $U = S^{v'} R_0^{m_0} R_r^{m_r} \mod n$.
+  Note that $m_0$ is always the user's secret key. This commitment is sent to
+  the issuer along with a [zero-knowledge proof](https://irma.app/docs/zkp/)
+  of not just $v'$ and $m_0$, but also over this share $m_{r}'$.
+
+- The issuer samples a random prime $e$. The issuer also samples $v''$ and $m_{r}''$ at random.
+  Since attribute values ($m$'s) are at most 256 bits, $m_{r}''$ is also 255 bits (to not overflow).
+  Next, the issuer computes 
+
+  $$
+    A = \Bigg(\frac{Z}{U S^{v''} R_{1}^{m_1} \dots \ R_{r}^{m_r''} \dots \ R_{n}^{m_n}}\Bigg)^{1/e}
+  $$
+
+  For every attribute that is tagged randomblind, the issuer includes its share
+  of the attribute in the exponent. In this case, only for index $r$.  For
+  regular attributes, the exponent is simply the attribute value that the
+  issuer wants to pass. The issuer sends $(A, e, v'', m_{r}'')$ back to the
+  user, again, along with a zero-knowledge proof of $A$.
+
+- Finally, the user is able to construct a valid CL-signature for the credential
+  by summing up the shares:
```

Comment:
In the formulas below the user constructs not a signature but the randomblind attributes; I would suggest you say that here instead.